### PR TITLE
chore(format): update to latest formatter

### PR DESCRIPTION
**Pull request by alexeagle**
- **Please check if the PR fulfills these requirements**
- [ X ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ X ] Tests for the changes have been added (for bug fixes / features)
- [ X ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by mprobst** [comment]
LGTM, though I'm not quite sure what to review here. The changes are mechanical and seem OK, right?

Are you missing the change to package.json. shinkwrap etc?

File: modules/angular2/src/common/forms/directives/ng_control_name.ts [hunk 1]
```diff
@@ -111,35 +92,37 @@ export class NgControlName extends NgControl implements OnChanges,
                   /* Array<Validator|Function> */ any[],
               @Optional() @Self() @Inject(NG_VALUE_ACCESSOR)
               valueAccessors: ControlValueAccessor[]) {
```
Comment (mprobst) [inline modules/angular2/src/common/forms/directives/ng_control_name.ts]: This looks like a bug though, could you file one?

Comment (alexeagle) [inline modules/angular2/src/common/forms/directives/ng_control_name.ts]: sure, but what is the bug? You'd expect valueAccessors to fit on the line above following the decorators?

**Comment by alexeagle** [comment]
shrinkwrap change is included (or the lint job would have failed https://travis-ci.org/angular/angular/jobs/121578928)
but I can't see how to get github to show the files, even in the search box it doesn't appear. I guess it's too many files for their UI...

File: modules/angular2/src/common/forms/directives/ng_control_name.ts (hunk 1 above)
Comment (mprobst) [inline modules/angular2/src/common/forms/directives/ng_control_name.ts]: indentation is way too high for the class body, isn't it?


>

Comment (alexeagle) [inline modules/angular2/src/common/forms/directives/ng_control_name.ts]: this indentation is still the constructor parameters.
class body starts with `super();` below and looks fine...

Comment (mprobst) [inline modules/angular2/src/common/forms/directives/ng_control_name.ts]: I think `ngOnChanges` should be indented +4 (2 for class, 2 for method), but it's on what looks like +10?

**Pull request closed without merging**